### ISSUE TITLE
Get people to fill out demographic fields for challenge

### ISSUE THREAD

**Issue by cheerfulstoic** [neo4j-examples/graphgist_portal#47]

**Comment by ryguyrg** [comment]
what demographic fields?  we no longer need address for swag fulfillment.

**Comment by cheerfulstoic** [comment]
You're right about that.  Though I think we should make sure that some means of contacting them (E-Mail or Twitter) is there in case they're a winner of the contest

**Comment by whatSocks** [comment]
Since I was asked to stalk a bunch of people to find their e-mail addresses, optional contact e-mail for *at least* challenge entries would be awesome. This would also help with consolidating accounts for people that create accounts with github, twitter, and (in the future, neo technology gmail addresses)
